src/constants.ts

```diff
@@ -25,6 +25,7 @@
 /** @hidden */
 export const acl = {
```

Comment:
We could discuss this, but the current naming convention we have for generated vocabs is to upper-case them (i.e., so this would be ACL).

Reply:
@Vinnl I think there was a reason for this, but I'm not sure what it was.

Reply:
Don't think we explicitly discussed this, but they're not generated and there's no need for them to STAND OUT SCREAMING OVER THE OTHER CODE IMHO :)

Reply:
Being generated or not isn't the issue (and we are generating constants for all these values already, and publishing them to npmjs, but just not depending on those npmjs modules at the moment).
Anyway, upper-casing classes of constants has always been standard practice in Java, and has always seemed to make total sense there (i.e., I've never heard anyone complain about the screams :) !).
We could trivially switch to lower-case for JavaScript, but does it really have a best practice for 'classes containing constants'...??!? These class names will nearly always be the prefixes recommended by the vocabularies themselves, so they'll necessarily be short, very often acronyms, and not the most descriptive in themselves (e.g., FOAF, DCTerms, LDP, RDF, RDFS, ACL, WAC, etc.), so that seems (to me at least) to be further justification for leaving them as they are in uppercase (i.e., the screaming is actually justified).